src/js/button.js

```diff
@@ -26,24 +26,14 @@ class Button extends ClickableComponent {
    * @method createEl
    */
   createEl(tag = 'button', props = {}, attributes = {}) {
+    if (tag !== 'button') {
+      throw new Error('Buttons must be created with a "button" tag.');
```

Comment:
https://github.com/videojs/video.js/blob/master/src/js/tracks/text-track-display.js#L165
https://github.com/videojs/video.js/blob/master/src/js/control-bar/live-display.js#L38 
https://github.com/videojs/video.js/blob/master/src/js/control-bar/control-bar.js#L39
https://github.com/videojs/video.js/blob/master/src/js/loading-spinner.js#L19 
https://github.com/videojs/video.js/blob/master/src/js/menu/menu-button.js#L114
https://github.com/videojs/video.js/blob/master/src/js/popup/popup-button.js#L62
https://github.com/videojs/video.js/blob/master/src/js/player.js#L492

Most createEl calls that we have don't seem to follow this format (other than base classes) like clickable-component and component basically.